libs/msc-flash/msc-flash-harmony/src/lib/reducers/flashing.interface.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { FlashingProcessState } from "../constants"
+import { MscFlashDetails } from "../dto"
+
+export interface FlashingState {
+  processState: FlashingProcessState
+  mscFlashingFilesDetails?: MscFlashDetails
```

Comment:
```suggestion
  mscFlashDetails?: MscFlashDetails
```
Suggest renaming `mscFlashingFilesDetails` to `mscFlashDetails` to avoid confusion, as the files are part of the structure, and using the original abstraction is clearer.